Stake update font size

ffmcgee725 opened the pull request:
https://github.com/OlympusDAO/olympus-frontend/issues/310

updated font size and reduced space between "stake unstake".

ffmcgee725 committed:
updated font size and reduced space between 'stake unstake'.

ffmcgee725 committed:
removed local eslintrc.js config.

dwjanus reviewed (commented):
Since we use tabs in other areas of the app this shouldnt be isolated to a custom class style in staking. Can you modify the Tab object in global js ? Otherwise use a generic classname, add class styles to styles.scss and include that classname for any occurrences of tabs. (The former is much more preferred)

ffmcgee725 committed:
refactored MuiTab object in global.js to contain changes to it affects all Tabs across the app. Removed selector with changes in stake.scss.

ffmcgee725 commented:
> 
> 
> Since we use tabs in other areas of the app this shouldnt be isolated to a custom class style in staking. Can you modify the Tab object in global js ? Otherwise use a generic classname, add class styles to styles.scss and include that classname for any occurrences of tabs. (The former is much more preferred)

Done ohmie! Modified the MuiTab object in global.js and removed the previously implemented selector from stake.scss.
